client/src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components/macro'
+
+const MovieCard = ({ imgUrl, title, year, genre }) => {
+  return (
+    <Wrapper>
+      <img src={imgUrl} alt={title} height="562" width="375" />
+      <h2>{title}</h2>
```

Comment:
I would not add the `title` as an `alt`-text. The reason being that in the following line you are adding the title as well. Therefore the image is not crucial for users to understand. Right now a screen reader would read something along the line of: 
image Harry Potter, heading Harry Potter ...
We are telling the user twice that it is an image of Harry Potter which is kind of unnecessary. I would leave the `alt` attribute empty here.
```suggestion
      <img src={imgUrl} alt="" height="562" width="375" />
      <h2>{title}</h2>
```